pyscriptjs/src/components/pyenv.ts

```diff
@@ -60,9 +62,23 @@ export class PyEnv extends HTMLElement {
         }
 
         async function loadPaths() {
+            var self = this;
```

Comment:
same comment here. Is this used anywhere?

Reply:
Agreed - line removed.